extensions/shared/videopress-plan-check.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { addFilter } from '@wordpress/hooks';
+import { createHigherOrderComponent } from '@wordpress/compose';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import UpgradeNudge from './components/upgrade-nudge';
+
+const replaceMediaPlaceholder = createHigherOrderComponent(
+	OriginalPlaceholder => props => {
+		if ( 'wp-block-video' !== props.className ) {
```

Comment:
I think there might be a `name` prop for the block name (`core/video`)? (but I can't remember for sure). If so, that might be more stable than checking the the className.

Reply:
With this filter, I don't think there is. I had a look and couldn't see that prop available. There might be another way though.

Reply:
The only other possible way I can see is to piggyback off the existing VideoPress modification block.